ccd-definition/CaseTypeTab/CareSupervision/8_C2.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "C2Tab",
+    "TabLabel": "C2",
+    "TabDisplayOrder": 8,
```

Comment:
probably we should reindex other tabs. If 2 tabs have same display order, then real order I guess is unspecified and tabs can change order